src/Microsoft.DotNet.ProjectModel/Utilities/FrameworksExtensions.cs

```diff
@@ -56,5 +56,12 @@ public static string GetFrameworkString(this NuGetFramework self)
             }
             return name + "-" + frameworkName.Profile;
         }
+        
+        public static bool IsPackageBased(this NuGetFramework self)
```

Comment:
Maybe let's not make a new public method in DotNet.ProjectModel for it then, so we can remove it later.  Also, putting that as a comment linked to the Nuget issue would be nice so we can know when it gets fixed in NuGet we can remove it.